packages/design-system/src/molecules/footerV2/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { View } from 'native-base'
+import { useCallback } from 'react'
+import LogoIso from '../../assets/svg/logo-iso.svg'
+import { Caption } from '../../atoms'
+import {
+  discordInviteUrl,
+  faqUrl,
+  instaUrl,
+  resourceCenterUrl,
+  stakingTermsUrl,
+  twitterUrl,
+} from '../../constants'
+import { openLink } from '../../utils'
+
+interface FooterOptionProps {
+  title: string
+  isLast?: boolean
+  link: string
+}
+const FooterOption: React.FC<FooterOptionProps> = ({
+  title,
+  link,
+  isLast = false,
+}) => {
+  const handlePress = useCallback(() => {
+    openLink(link, true)
+  }, [link])
+  return (
+    <View
+      mr={{ md: !isLast ? '4' : 0 }}
+      mb={{ base: !isLast ? '4' : 0, md: 0 }}
+    >
+      <Caption weight="bold" color="rgba(0, 0, 0, 0.3)" onPress={handlePress}>
```

Comment:
I'll add colors in a different PR. These are the new colors we are going to use on the design system.